src/app/material.module.ts

```diff
@@ -0,0 +1,42 @@
+import {NgModule} from '@angular/core';
+import {
+  MdButtonModule,
+  MdCardModule,
+  MdDialogModule,
+  MdIconModule,
+  MdInputModule,
+  MdMenuModule,
+  MdOptionModule,
+  MdSidenavModule,
+  MdSlideToggleModule,
+  MdToolbarModule
+} from '@angular/material';
+
+@NgModule({
+  imports: [
+    MdButtonModule,
+    MdCardModule,
+    MdDialogModule,
+    MdIconModule,
+    MdInputModule,
+    MdMenuModule,
+    MdOptionModule,
+    MdSidenavModule,
+    MdSlideToggleModule,
+    MdToolbarModule
+  ],
+  exports: [
+    MdButtonModule,
+    MdCardModule,
+    MdDialogModule,
+    MdIconModule,
+    MdInputModule,
+    MdMenuModule,
+    MdOptionModule,
```

Comment:
Hmm this is weird. It should be exported by the `MdMenuModule` ([here](https://github.com/angular/material2/blob/master/src/lib/select/index.ts#L22)).

I'd be fine keeping it for now and we can revisit this later again.